client/post-editor/media-modal/markup.js

```diff
@@ -60,9 +60,8 @@ const Markup = {
 	 * Adapted from WordPress.
 	 *
 	 * @copyright 2015 by the WordPress contributors.
-	 * @license See CREDITS.md.
+	 * @license LGPL-2.1
```

Comment:
Is it LGPL or GPL? I'm by no means an expert in that area, I just don't find any reference in the WordPress source which says it's the _Lesser_ version of the license.

Reply:
I'm not sure. I went to the URL linked in line 64, traversed it up until I found a license.txt (https://github.com/WordPress/WordPress/blob/4.3/wp-includes/js/tinymce/license.txt), and searched for the name in https://spdx.org/licenses/

Reply:
Ah, TinyMCE seems to be in fact LGPL. I got confused from the _Adapted from WordPress_ two lines above.